BankingSystem/BankingSystem/Program.cs

```diff
@@ -0,0 +1,380 @@
+                Console.WriteLine("\t MENU: ");
+                Console.WriteLine("1. Create account");
+                Console.WriteLine("2. Domestic transfer");
+                Console.WriteLine("3. Outgoing transfer");
+                Console.WriteLine("4. Accounts balance");
+                Console.WriteLine("5. Transfer History");
+                Console.WriteLine("6. Exit");
+                Console.WriteLine();
+
+                int userOption = GetIntFromUser("** Select option by number: **");
+
+                switch (userOption)
+                {
+                    case 1:
+                        UserWantsToCreateAccount();
+                        break;
+                    case 2:
+                        DomesticTransfer();
+                        break;
+                    case 3:
+                        OutgoingTransfer();
+                        break;
+                    case 4:
+                        ShowAllOwnAccounts();
+                        break;
+                    case 5:
+                        TransferHistory();
+                        break;
+                    case 6:
+                        Exit();
+                        break;
+                    default:
+                        Console.WriteLine();
+                        Console.WriteLine("!!!!!!!!!!!!!!!!!");
+                        Console.WriteLine("Unknown option.");
+                        Console.WriteLine("!!!!!!!!!!!!!!!!!");
+                        break;
+                }
+
+            } while (true);
+        }
+
+        void UserWantsToCreateAccount()
+        {
+            var newAccountName = "";
+            do
+            {
+                newAccountName = GetUserInputBasedOnMessage("** Enter your account name: **");
+                bool exist = _accountsAdmin.CheckIfAccountExistByName(newAccountName);
+
+                if (exist != true)
+                {
+                    break;
+                }
+                else
+                {
+                    Console.WriteLine();
+                    Console.WriteLine("!!!!!!!!!!!!!!!!!");
+                    Console.WriteLine("Account with that name exists.");
+                    Console.WriteLine("!!!!!!!!!!!!!!!!!");
+                }
+
+            } while (true);
+
+
+            float newAccountValue = 1000;
+
+            var account = new Account(newAccountName, newAccountValue);
+
+            Console.WriteLine();
+            Console.WriteLine("\t Your account details :");
+            Console.WriteLine("-----------------------------");
+            Console.WriteLine("Name Account: " + account.AccountName);
+            Console.WriteLine("Number account: " + account.AccountNumber);
+            Console.WriteLine("Value: " + "{0:C} ", account.AccountValue);
+            Console.WriteLine("-----------------------------");
+            _accountsAdmin.AddNewAccountToList(account);
+
+        }
+
+        void DomesticTransfer()
```

Comment:
The method can be simplified. In contains two equal block of choosing and account from the list. Also,, the modification of the balance of the accounts can be shared. The very same methods can then be reused by OutgoingTransfer logic.